main_logic/tts_client.py

```
@@ -204,6 +189,163 @@ def _get_tts_language_code() -> str:
+        nonlocal synth_task
+        if synth_task is not None and not synth_task.done():
+            return
+        if not pending:
+            synth_task = None
+            return
+        text, sid = pending.pop(0)
+
+        async def _do():
+            try:
+                await synthesize_fn(text, sid)
+            except Exception as exc:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)